src/analyze/analyze-time-data.h

```diff
@@ -37,9 +37,22 @@ typedef struct BootTimes {
         usec_t reverse_offset;
 } BootTimes;
 
+typedef struct RelatedUnits {
```

Comment:
I do not think this is necessary. We can use `enum UnitDependency` and `unit_dependency_to_string()`. How about the following?
```c
typedef struct UnitTimes {
        ...
        char **deps[_UNIT_DEPENDENCY_MAX];
        ...
} UnitTimes;
```
Then, 
```c
static void plot_tooltip(const char *name, const UnitTimes *u) {
        ...
        VA_ARGS_FOREACH(i, UNIT_AFTER, UNIT_BEFORE, UNIT_CONFLICTS, ...)
                if (!strv_isempty(t->deps[i])) {
                        svg("\n%s:\n", unit_dependency_to_string(i));
                        STRV_FOREACH(s, t->deps[i])
                                svg("  %s\n", *s);
                }
}
```